duneanaobj/StandardRecord/SRGArTrack.h

```diff
@@ -0,0 +1,52 @@
+////////////////////////////////////////////////////////////////////////
+// \file    SRGArTrack.h
+// \brief   Reconstructed GAr track object
+// \author  F. Martinez <f.martinezlopez@qmul.ac.uk>
+// \date    Jan. 2023
+////////////////////////////////////////////////////////////////////////
+#ifndef DUNEANAOBJ_SRGARTRACK_H
+#define DUNEANAOBJ_SRGARTRACK_H
+
+#include "duneanaobj/StandardRecord/SRVector3D.h"
+#include "duneanaobj/StandardRecord/SRParticleTruth.h"
+
+namespace caf
+{
+  class SRGArTrack
+  {
+    public:
+      SRVector3D start;      ///< Track 3D start point
+      SRVector3D end;        ///< Track 3D end point
+      SRVector3D dir;        ///< Unit vector representing estimate of track direction *taken from start point*
+      SRVector3D enddir;     ///< Unit vector representing estimate of track direction *taken from endpoint*
+
+      // Track characteristics
+      float dEdx_F = -999.;  ///< dE/dx of track in forward fit
+      float dEdx_B = -999.;  ///< dE/dx of track in backward fit
+
+      float p_F = -999.;     ///< Reco momentum of track in forward fit
+      float p_B = -999.;     ///< Reco momentum of track in forward fit
+
+      float len_cm_F = -999.; ///< Length of track in forward fit
+      float len_cm_B = -999.; ///< Length of track in backward fit
```

Comment:
I think `_fwd` and `_bkwd` are probably preferable in these names.  I appreciate that you've documented them here---but from experience I bet future new users will appreciate knowing what they are without having to work all the way back to the documentation so often!  (The `_B` one especially makes me think of magnetic fields every time I look at it...)  😄 

```suggestion
      float dEdx_fwd = -999.;  ///< dE/dx of track in forward fit
      float dEdx_bkwd = -999.;  ///< dE/dx of track in backward fit

      float p_fwd = -999.;     ///< Reco momentum of track in forward fit
      float p__bkwd = -999.;     ///< Reco momentum of track in forward fit

      float len_cm_fwd  = -999.; ///< Length of track in forward fit
      float len_cm__bkwd = -999.; ///< Length of track in backward fit
```